benefits.md

```diff
@@ -0,0 +1,43 @@
+# Benefits & Perks
+
+This page details the benefits of working at 2i2c.
+
+## Health Benefits
+
+For employees based in the US, 2i2c provides generous health benefits. Health benefits at 2i2c are provided through its parent organization, {term}`ICSI`. For more information, see [the ICSI health benefits page](https://drive.google.com/file/d/1lm16SjCA1SbkgzDpcfg3yEKCB6eDOkx7/view?usp=sharing).
+
+For employees who are not based in the US, we will work to offer similar benefits that apply to your country of residence.
+
+## Retirement
+
+For employees based in the US, 2i2c will make contributions to a retirement account. See [the ICSI retirement benefits page](https://drive.google.com/file/d/1Of323rUco7TvbkavgCUtBji4INdX-TJB/view?usp=sharing) for more information.
+
+## Paid Time Off
+
+First and foremost, 2i2c fosters a culture of healthy balance between work and life.
+All 2i2c employees should take the time they need to thrive - this will both make you happier and benefit 2i2c in the long run.
+
+Currently 2i2c does not have a limit on the number of days you may take off each year. We encourage a **don't ask, do tell** policy to time off:
```

Comment:
I really like https://buffer.com/resources/employees-take-vacation/. Primary problem is that when folks don't know how much vacation others are taking, it causes them to take less time off than they 'should'. 

The primary additions here would be the suggested minimum vacation days policy. I like buffer's example of 15 workdays (~3 weeks). 

I also like the other aspects of their policy, which I've cribbed below.

---
Here are the exact guidelines we sent to the team:

1. We encourage team members to take a minimum of 3 weeks (15 work days) of vacation time throughout the year, in addition to the holidays (bank, religious or otherwise) you choose to observe.
2. There is no maximum vacation recommendation, though there may be more and longer conversations for unique situations in which a teammate might want to take, say, 6–8 weeks off.
3. Buffer is privileged to be a global team; please take time off that’s in line with your country’s standards if it’s generally a bit more than our recommendation.
4. It’s generally great to begin to take time off after you’ve been at Buffer for about 3 months. (If you’re newer to the team and have something already planned, no worries! We’ll take it case by case).
5. This time is separate from holidays that you choose to celebrate, and outside of sick/personal days, bereavement/compassionate leave, and family leave.

Reply:
Not sure how to make this interact with the 14day winter holiday though. I also know many people (including myself!) don't like going places or doing traditional things around the winter break. I used to always just trade that for more time in February.

Reply:
I like the buffer docs on this as well, we should include them as inspiration for this. I suspect that this policy might evolve over time, so I wonder what we can use as an MVP right now and then open issues to discuss the finer points later (since we don't have *any* process right now, it seems like just having something would be an improvement as long as it is not really controversial).

Also just a note that the 2 weeks at the end-of-year thing is just us inheriting an ICSI policy...I'm not sure how that works but my assumption is that we should assume that our policy is constrained by ICSI policy to some degree unless we have an official agreement otherwise? Either way - something we can continue to iterate on

Reply:
Yeah, this sounds good to me.